src/showTodo.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import TodoListMain from './todoListMain'
+
+const ShowTodo = (props) => {
+  return (
+    <div>
+      <center>
+      <h1>Todo List</h1>
+        <hr />
+        <ul>
+          {
```

Comment:
I prefer not to break lines inside of string interpolation blocks. Just adds white space that's not needed :)